meson/postconf.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+import os
+import shutil
+import subprocess
+
+base_dir = os.environ.get('MESON_SOURCE_DIR', '.')
+
+STEPS = 12
+IMAGES_PER_STEP = 4
+SIZE=38
```

Comment:
```suggestion
SIZE = 38
```
...just for consistency.

If there is a source for where these values are coming from (plymouth docs/source or whatever) then I think a link would also be good in case these need to be updated in the future.

Reply:
@jlnr thanks for your feedback.

I couldn't find any sources for it, we have taken the size of Ubuntu's theme.